app/cmd/root.go

```diff
@@ -93,6 +93,21 @@ func init() {
 	RootCmd.PersistentFlags().IntVar(&config.API.GrpcServerPort, "grpc-port",
 		config.API.GrpcServerPort, "GRPC api server port")
 
+	/**========================Hare Flags ========================== **/
+
+	// N determines the size of the hare committee
+	RootCmd.PersistentFlags().IntVar(&config.HARE.N, "hare-committee-size",
+		config.API.JSONServerPort, "Size of Hare committee")
```

Comment:
you need to change `config.API.JSONServerPort` to `config.HARE.N`. also do this to all the other commands you added.

Reply:
Fixed